.yaydoc.yml

```diff
@@ -0,0 +1,19 @@
+metadata:
+  author: FOSSASIA
+  projectname: "Open Event"
+  version: development
+build:
+  theme:
+    name: sphinx_fossasia_theme
+  source: dev-handbook
```

Comment:
It has been moved to the `docs` folder to make it similar to other projects. Could you update things accordingly ?